src/client/styles.css

```
@@ -6,6 +6,7 @@
 @theme {
   --default-ring-width: 3px;
   --default-ring-color: var(--color-malibu-blue);
+  --font-display: "OpenFront", Inter, Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)